src/components/reportingCards/ReportingCard8.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ReportingCard from './ReportingCard';
+import useLoadData from './ReportingCard.hook';
+import PeopleIcon from '@material-ui/icons/People';
+
+export default function component(props) {
+  const { startDate, endDate, disableSeeMore, rows } = props;
+
+  const data = useLoadData(
+    startDate,
+    endDate,
+    'gender_details',
+    'gender_details',
+    (data) =>
+      data.gender_details.reduce((acc, cur) => acc + Number(cur.number), 0),
+    rows
+  );
+
+  const formatData = (data) => {
+    const result = data.top.map((item) => {
+      return {
+        name:
+          item.name && item.name.match(/f/i)
+            ? 'Female'
+            : item.name
+            ? 'Male'
+            : 'Unknown',
+        num: `${item.percentage}%`,
+      };
+    });
+    return { ...data, top: result };
+  };
+
+  return (
+    <ReportingCard
+      text={{
+        title: 'Grower Gender Count',
+        text1: 'Total growers',
```

Comment:
Looks like there's no total in the data from the API. I'm not sure how useful it would be anyway.

Reply:
I agreed before, but now I think the total does end up being useful for interpreting the numbers of each gender.  

I added the total to the api data and made the gender_details more consistent with the other report cards' data now.  It saves us working around it on the client.

Reply:
Percentages make more sense to me, but the format isn't great -- What do you think?

Reply:
Another option is to show both values with a separator, but we'd need to be careful with alignment.

Let's stick with raw numbers for now for consistency. It's easy for a user to convert numbers to percentages.